src/UI/Icon.elm

```diff
@@ -38,8 +44,17 @@ type alias Properties =
     }
 
 
+type alias Options =
+    { color : IconColor }
+
+
+type IconColor
+    = ColorFromPalette Palette.Color
+    | ColorInherit
```

Comment:
I would leave the black color as default instead of having a color Inherited (same for texts btw)

Reply:
This can be done, but it will require another PR, fixing the color everywhere icons are used in WMS.